packages/browserify/examples/00-simple-cli/index.html

```diff
@@ -1,10 +1,10 @@
-<!DOCTYPE HTML>
```

Comment:
(could) examples are ignored by eslint and I think we should consider using the same ignore rules for prettier.